### PR TITLE
Misc cleaning based on the changes done to EBML to get a proper RFC

### PR DESCRIPTION
**Pull request by robUx4**

### CODE REVIEW

File: bootstrap [hunk 1]
```diff
@@ -31,7 +31,7 @@ check_version() {
         echo "$1 too old (got $gotver, needed $3)"
         NEEDED="$NEEDED $1"
     else
-        FOUND="$FOUND $1"
+        FOUND="$FOUND $(shell which $1)"
```
[the remark this answers is not in the document]
Comment (retokromer) [reply bootstrap:34]: FYI: `command -v` works fine on macOS, Linux and Windows Terminal on my end.

[the remark this answers is not in the document]
Comment (robUx4) [reply bootstrap:34]: I have no way to test a fix so it's better to add a patch that works for you on top of this branch.

In VLC we use `$(which)` in Makefiles and it seems to work.

Also is it `$(shell command -v $1)` or `$(command -v $1)` ?

[the remark this answers is not in the document]
Comment (retokromer) [reply bootstrap:34]: I use `$(command -v $1)` in my cross-platform Makefiles.

[the remark this answers is not in the document]
Comment (robUx4) [reply bootstrap]: Thanks, I added a patch that should fix this.

**Pull request merged**